code/procs/accents.dm

```diff
@@ -868,6 +921,75 @@ proc/random_accent()
 		T.update()
 	return modded
 
+/proc/germify(var/string) // pretty much the same thing as scots and tyke, but instead with some common cognates between english and german. The list is significantly smaller
+
+	var/list/phrase = list(
+		"excuse me" = "entschuldigung",
+		"yes sir" = "javol",
+		"yes maam" = "javol",
+		"yes ma'am" = "javol",
+		"good morning" = "guten morgen",
+		"good day" = "guten tag",
+		"good afternoon" = "guten tag",
+		"good evening" = "guten abend",
+		"good night" = "guten nacht",
+		"thank you" = "danke",
+		"that's too bad" = "schade",
+		"thats too bad" = "schade",
+		"too bad" = "schade",
+		"no problem" = "kein problem"
+	)
```

Comment:
why have this in addition to `german.txt`?

Reply:
I could also rework the system itself, but I want to keep the phrases and the single word replacements seperate

Reply:
worth adding that as a comment at least

Reply:
alrighty. I'll try and comment the code a bit better